utils/src/utils/AzExtFsExtra.ts

```diff
@@ -4,6 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { dirname } from 'path';
+import path = require('path');
```

Comment:
I'd merge this with the above and do `import * as path from 'path';`, and update where `dirname` gets used.